api/controllers/track.js

```diff
@@ -1,20 +1,25 @@
 // function: multipleTracks
 // if no specified album id, sends all tracks in the database
 // if specified album id, returns all tracks in the given album
-const multipleTracks = async (req, res) => {
+export async function multipleTracks(req, res) {
   let conn;
   let q;
-  const albumid = req.query.albumid;
+  const albumid = parseInt(req.query.albumid, 10);
+  console.log(albumid);
+  let result;
   try {
     conn = await req.app.locals.pool.getConnection();
-    if(albumid) {
-      q = 'SELECT * FROM `sikhifm_db`.`TrackAlbum` LEFT JOIN `sikhifm_db`.`Track`'+
-       'ON `TrackAlbum`.`Album` = `Track`.`ID` WHERE `TrackAlbum`.`Album` =' + albumid + ';';
+    if (albumid) {
+      q =
+        'SELECT * FROM `TrackAlbum` LEFT JOIN `Track`' +
```

Comment:
one improvement would be instead of `Select *` is to list the exact columns you want from the table. It's better practice so only the relevant information is sent.

Reply:
I forgot to ask which columns you think are relevant, but yeah, It shouldn't be a difficult change.